### PR TITLE
Auto Rehost for debugger Usage

### PR DESCRIPTION
**Pull request by NikoCat233**
This feature may be back to InnerSloth's Policy
So only limit to debug usage

Only functions as host disconnect and not ExitGame
Use keybind LeftShift + C to cancel a rehost countdown
Also U can cancel host side auto play again with LeftShift + C

### CODE REVIEW

**Comment by Ultradragon005** [comment]
mega OP OP

**Comment by Tommy-XL** [comment]
I think it's better to leave this for the next dev version

**Review by Tommy-XL — APPROVED** [review]

**Comment by 0xDrMoe** [comment]
Is this ready?

File: Patches/ClientOptionsPatch.cs [hunk 1]
```diff
@@ -23,8 +23,9 @@ public static class OptionsMenuBehaviourStartPatch
     private static ClientOptionItem SwitchVanilla;
 #if DEBUG
     private static ClientOptionItem VersionCheat;
+    private static ClientOptionItem GodMode;
+    private static ClientOptionItem AutoRehost;
 #endif
```
Comment (Tommy-XL) [inline Patches/ClientOptionsPatch.cs:28]: If dev versions are built as debug, then these settings will appear for users of the dev version

Comment (NikoCat233) [reply Patches/ClientOptionsPatch.cs:28]: I dont think we are building debug...

**Pull request merged**